### DISCUSSION TITLE
Riverpod extension for VSCode

### DISCUSSION THREAD

**Discussion by Trung15010802** [rrousselGit/riverpod#3757] [Show and tell]
![image](https://github.com/user-attachments/assets/1bc11b34-9db6-43ee-bd67-1fee35f31238)
I am someone who knows nothing about TypeScript and programming extensions for VSCode. I worked on this for 2 days, so this repository is not perfect.

Please create a Pull Request if you can help me improve it!

**Comment by rrousselGit** [comment]
Note that Riverpod already provides such refactor, using riverpod-lint

**Reply by Trung15010802** [reply]
Yeah. My extension isn't as good as riverpod_lint. But sometimes riverpod_lint doesn't work (I have to activate it again), and I don't want to import custom_lint and riverpod_lint just to use the right context menu in all of my projects
